examples/chain/0_LocalOrderHash/example.go

```diff
@@ -43,19 +45,31 @@ func main() {
 	)
 
 	if err != nil {
-		fmt.Println(err)
+		panic(err)
 	}
 
 	clientCtx = clientCtx.WithNodeURI(network.TmEndpoint).WithClient(tmClient)
 
-	chainClient, err := chainclient.NewChainClient(
+	exchangeClient, err := exchangeclient.NewExchangeClient(network)
+	if err != nil {
+		panic(err)
+	}
+
+	ctx := context.Background()
+	marketsAssistant, err := core.NewMarketsAssistantUsingExchangeClient(ctx, exchangeClient)
```

Comment:
I meant we can do something like 

```
NewExchangeClient():
   initializeDefaultAssistants

func (c *ExchangeClient) WithAssistant(someThing) *ExchangeClient:
   c.assistant = someThing
  return c
```

This way MM when use sdk, they don't really need to change their code

Reply:
We could, but I did not think it would be a good idea to make the ChainClient know the ExchangeClient. Connecting those two components will produce a coupling that, by the current design, I don't think should exist really.